packages/protocol/contracts/governance/Governance.sol

```diff
@@ -666,7 +692,64 @@ contract Governance is IGovernance, Ownable, Initializable, ReentrancyGuard, Usi
   }
 
   /**
-   * @notice Withdraws refunded Celo Gold commitments.
+   * @notice Whitelists the hash of a hotfix transaction(s).
+   * @param txHash The abi encoded keccak256 hash of the hotfix transaction(s) to be whitelisted.
+   */
+  function whitelistHotfix(bytes32 txHash) external {
+    hotfixes[txHash].whitelisted[msg.sender] = true;
+    emit HotfixWhitelisted(txHash, msg.sender);
```

Comment:
https://github.com/celo-org/celo-monorepo/pull/613/files#r334826047

+1 to this